_includes/tenpillars.html

```diff
@@ -677,34 +674,34 @@ <h2>Example Checklist</h2>
     <tr>
         <td>Standardized</td>
         <td>4</td>
-        <td>Need golden paths for Kubernetes, Lambda, and VM deployments</td>
+        <td>Need golden paths for Kubernetes, Lambda, and VM deployments.</td>
     </tr>
     <tr>
         <td>Maintainable</td>
```

Comment:
@mcasperson Maintainable seems to be doubled up in the table, but the second entry has an extra line:
Smoke test script to hit API, web page, and database login.
Guessing one of the rows should be deleted but wanted to confirm with you first.

Reply:
No problem, I can fix that up.